src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Unix.cs

```diff
@@ -18,48 +18,58 @@
     {
         private const string DefaultTimeZoneDirectory = "/usr/share/zoneinfo/";
 
-        // UTC aliases per https://github.com/unicode-org/cldr/blob/master/common/bcp47/timezone.xml
+        // Set fallback values using abbreviations, base offset, and id
+        // These are expected in environments without time zone globalization data
+        private string? s_standardAbbrevName;
```

Comment:
> s_s [](http://example.com/codeflow?start=23&length=4)

we only start variables with `_s` when it is static. please change that.

Reply:
Thanks! I originally put them as static but realized I didn't want this to be shared across all TimeZoneInfo objects, so removed the keyword without changing the name!